raiden-ts/tests/unit/states/4_full.json

```diff
@@ -1,6 +1,6 @@
 {
   "address": "0x11111111111111111111111111aaaaaaaaaaaaaa",
-  "version": 3,
+  "version": 4,
```

Comment:
Not mistaken, actually the previous version, when the 4th migration was applied, resulted in a valid 4th-version state, so tests passed and did it correctly, it's just house-keeping here, because they're intended to be about the 4th version migration.